### PR TITLE
CNV-39918: Improve nad list page

### PR DESCRIPTION
**Pull request by avivtur**
1. Replace useHistory with useNavigate (React router v5 to v6)
2. Using Empty state component
3. Adjusting row styling to fit entire row
4. Changing create button to dropdown to expose create from YAML without entering the form first.

### CODE REVIEW

File: src/views/nads/list/components/NADListEmpty.tsx [hunk 1]
```diff
@@ -42,27 +45,15 @@ const NADListEmpty: FC = () => {
         titleText={<>{t('No NetworkAttachmentDefinition found')}</>}
```
Comment (metalice) [inline src/views/nads/list/components/NADListEmpty.tsx:45]: need the fragment?

File: src/views/nads/list/components/NADCreateDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+type NADCreateDropdownProps = {
+  namespace: string;
+};
+
+const NADCreateDropdown: FC<NADCreateDropdownProps> = ({ namespace }) => {
+  const { t } = useNetworkingTranslation();
+  const navigate = useNavigate();
+
+  const createItems = {
```
Comment (metalice) [inline src/views/nads/list/components/NADCreateDropdown.tsx:17]: can be in util

File: src/views/nads/list/components/NADListEmpty.tsx (hunk 1 above)
Comment (avivtur) [reply src/views/nads/list/components/NADListEmpty.tsx]: No removed thanks

**Pull request merged**